documentation/projects.md

```diff
@@ -0,0 +1,114 @@
+# 🧪 Bolt projects
+
+🧪 **Experimental:** Bolt projects are experimental and may change in future
+versions of Bolt. Make sure you've updated to the most recent version of Bolt to
+take full advantage of this feature.
+
+One of the first things you'll do when you start using Bolt is create a Bolt
+project. A Bolt project is a simple directory that serves as the launching point
+for Bolt. You run Bolt from your project directory, and the directory houses
+your Bolt configuration files and your Bolt content, such as plans and tasks.
+
+In addition to working with your local Bolt content, Bolt projects give you a
+way to share that content with other users in your organization. You can create
+orchestration that is specific to the infrastructure you're working with, and
+then commit the project directory to git for others to consume.
+
+Here is an example of a typical project with a task, a plan, and an inventory file:
```

Comment:
Should task, plan, and inventory link out to docs about what those are?

Reply:
Let's add a related info section with links to those docs